topological_navigation/scripts/localisation.py

```diff
@@ -161,6 +163,27 @@ def Callback(self, msg, item):
 
 
 
+    def get_nodes_wtag_cb(self,req):
+        tlist = []
+        rlist=[]
+
+        rospy.wait_for_service('/topological_map_manager/get_tagged_nodes')
+        try:
+            cont = rospy.ServiceProxy('/topological_map_manager/get_tagged_nodes', strands_navigation_msgs.srv.GetTaggedNodes)
+            resp1 = cont(req.tag)
+            tagnodes = resp1.nodes
+        except rospy.ServiceException, e:
+            print "Service call failed: %s"%e
```

Comment:
sorry to be a pain... but please always use `rospy.log...` for all of this. We are collecting logs and if you just print such a big error and not log it, we won't see it. This is a general rule...